INSTALLING.rst

```diff
@@ -0,0 +1,271 @@
+    - Python >= 2.7
+    - NumPy >= 1.7
+    - CMake >= 2.8.0
+    - C++11 capable compiler (tested with ``gcc`` 4.8, 4.9, 5.4, 6.4, 7.0,
+      8.0, ``clang`` 5.0, 6.0)
+- Optional:
+    - NVIDIA CUDA Toolkit >= 8.0
+    - Intel Threading Building Blocks >= 4.3
+    - MPI (tested with OpenMPI, MVAPICH)
+    - LLVM >= 3.6, <= 7.0.0
+- Useful developer tools
+    - Doxygen >= 1.8.5
+
+Software prerequisites on clusters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Most cluster administrators provide versions of Git, Python, NumPy, MPI, and
+CUDA as modules. You will need to consult the documentation or ask the system
+administrators for instructions to load the appropriate modules.
+
+Prerequisites on workstations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On a workstation, use the system's package manager to install all of the
+prerequisites. Some Linux distributions separate ``-dev`` and normal packages,
+you need the development packages to build **HOOMD-blue**.
+
+On macOS systems, you can use `MacPorts <https://www.macports.org/>`_ or
+`Homebrew <https://brew.sh/>`_ to install prerequisites. You will need to
+install Xcode (free) through the Mac App Store to supply the C++ compiler.
+
+Installing prerequisites with conda
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. caution::
+
+    *Not recommended.* Conda is very useful as a delivery platform for `stable
+    binaries <http://glotzerlab.engin.umich.edu/hoomd-blue/download.html>`_,
+    but there are many pitfalls when using it to provide development
+    prerequisites.
+
+Despite this warning, many users wish to use conda to provide those development
+prerequisites. There are a few additional steps required to build
+**HOOMD-blue** against a conda software stack, as you must ensure that all
+libraries (MPI, Python, etc.) are linked from the conda environment. First,
+install `miniconda <https://docs.conda.io/en/latest/miniconda.html>`_.
+Then, uninstall the ``hoomd`` package if it is installed,
+and install the prerequisite libraries and tools::
+
+    # if using linux
+    conda install sphinx git mpich2 numpy cmake pkg-config
+    # if using mac
+    conda install sphinx git numpy cmake pkg-config
+
+Check the CMake configuration to ensure that it finds Python, NumPy, and MPI
+from within the conda installation. If any of these library or include files
+reference directories other than your conda environment, you will need to set
+the appropriate setting for ``PYTHON_EXECUTABLE``, etc.
+
+.. note::
+
+    The ``mpich2`` package is not available on macOS. Without it,
+    **HOOMD-blue** will build without MPI support.
+
+.. warning::
+
+    On macOS, installing gcc with conda is not sufficient to build
+    **HOOMD-blue**. Update Xcode to the latest version using the Mac App
+    Store.
+
+.. _compile-hoomd:
+
+Compile HOOMD-blue
+------------------
+
+Set the environment variable ``SOFTWARE_ROOT`` to the location you wish to
+install **HOOMD-blue**::
+
+    $ export SOFTWARE_ROOT=/path/to/prefix
```

Comment:
@joaander I find this section confusing, and I'd like your thoughts on this potential revision:

Instead of suggesting an arbitrary `SOFTWARE_ROOT`, we could suggest users to pick a `site-packages` directory from `python -c "import site; print(site.getsitepackages())"` or `python -m site --user-site`. The current directions to use `$SOFTWARE_ROOT/lib/python` are likely incorrect for any "real" Python installation, in my understanding. A real installation path might look more like `$SOFTWARE_ROOT/lib/python3.7/site-packages`. In particular, we shouldn't install to `$SOFTWARE_ROOT/lib/python` (above `site-packages`) because that might put HOOMD alongside the libraries included with Python (e.g. `os`, `sys`), at least in my conda environment.